webapp/views/App/views/Designer/TaxonomyList/HeaderLeft/HeaderLeft.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { appModuleUri, designerModules } from '@webapp/app/appModules'
+
+import { matchPath, useHistory, useLocation } from 'react-router'
+import * as TaxonomyActions from '@webapp/loggedin/surveyViews/taxonomy/actions'
+
+import * as Taxonomy from '@core/survey/taxonomy'
+
+import { useI18n } from '@webapp/store/system'
+
+const HeaderLeft = () => {
+  const i18n = useI18n()
+  const dispatch = useDispatch()
+  const { pathname } = useLocation()
+  const history = useHistory()
+  const inTaxonomiesPath = Boolean(matchPath(pathname, appModuleUri(designerModules.taxonomies)))
```

Comment:
this check appears multiple times, can it be done only once?

Reply:
what do you think that can be the right place?

Reply:
I think it can be done once in TaxonomyView and passed as parameter to the table... I don't know actually if we need to pass it using another prop like headerProps... what do you think?!